README.md

```diff
@@ -308,7 +308,7 @@ If you want to revise the default configuration for kube-prometheus, for example
 
 #### Kubernetes Deployment
 
-To deploy this project, you can simply run `kubectl apply -f examples/standard` and a Kubernetes service and deployment will be created. (Note: Adjust the apiVersion of some resource if your kubernetes cluster's version is not 1.8+, check the yaml file for more information).
+To deploy this project, you can simply run `kubectl apply -k github.com/kubernetes/kube-state-metrics/` and a Kubernetes service and deployment will be created. (Note: Adjust the apiVersion of some resource if your kubernetes cluster's version is not 1.8+, check the yaml file for more information).
```

Comment:
I would still prefer to keep the default instruction without kustomize.

We can have another sentence after that that explains how to install with kustomize.

Reply:
You make the call. Then we can close this PR. You may just commit directly as you like.